sdk/src/androidTest/java/com/desk/android/sdk/util/ViewActions.java

```diff
@@ -0,0 +1,151 @@
+package com.desk.android.sdk.util;
```

Comment:
Looks like this class was added but is unused. Was there a specific reason for adding it?